composables/users.ts

```diff
@@ -101,19 +100,13 @@ if (process.client) {
       window.addEventListener('visibilitychange', windowReload, { capture: true })
     }
     // when sign out
-    else if (oldId) {
-      const oldUser = users.value.find(user => user.account?.id === oldId)
+    else if (oldHandle) {
+      const oldUser = users.value.find(user => user.account?.acct === oldHandle)
       // when sign out, the other tab will not have the user, idb is not reactive
       if (oldUser)
         window.addEventListener('visibilitychange', windowReload, { capture: true })
     }
   }, { immediate: true, flush: 'post' })
-
-  // for injected script to read
-  const currentUserHandle = computed(() => currentUser.value?.account.acct || '')
-  watchEffect(() => {
-    localStorage.setItem(STORAGE_KEY_CURRENT_USER_HANDLE, currentUserHandle.value)
-  })
```

Comment:
I believe this is still needed for https://github.com/elk-zone/elk/blob/1a577343daabfcb5dd1c45fd1ee630d464a00bb3/plugins/setup-head-script.server.ts#L11-L16 to pick up...

Reply:
Isn't this (updated) line sufficient?
```ts
const currentUserHandle = useLocalStorage<string>(STORAGE_KEY_CURRENT_USER_HANDLE, mock ? mock.user.account.id : '')
```

When I switch between accounts, it updates the `elk-current-user-handle` local storage value.

Reply:
Oh, you're quite right!